### ISSUE TITLE
Process "Left_Down",  "Left_Up" etc

### ISSUE THREAD

**Issue by vredesbyyrd** [Coffee2CodeNL/gebaar-libinput#2]
This tool looks great. Very happy to see a cleanly written alternative like this. Thanks for sharing :) 

I translated my _libinput-gestures.conf_ to the _gebaard.toml_ format and this is what I got:  

```
[commands.swipe.three]
up = "skippy-xd"
down = "/home/clu/Scripts/hotkeys/bsh_libinput-gestures-min-window.sh"
left = "xdotool key alt+Right"
right = "xdotool key alt+Left" 
right_up = "xdotool key control+t" 
right_down = "xdotool key control+w" 
left_down = "/home/clu/Scripts/hotkeys/bsh_libinput-gestures-close-window.sh"

[commands.swipe.four]
up = "xdotool key super+w"
down = "xdotool key super+q"
``` 

Everything works great except the "multi-directional" gestures e.g  `left_down, right_down`  etc are not recognized. Are these operational and my formatting is just wrong? If not, do you intend to add these gestures at some point? 

Thanks for your time.

**Comment by Coffee2CodeNL** [comment]
I will see if I can add the oblique variants! 😁

For now it's only up/down/left/right

**Comment by vredesbyyrd** [comment]
That would be rad  😻  They are just so drilled into my muscle memory it would be tough to unlearn at this point. 

Thanks for looking into it! Cheers.

**Comment by Coffee2CodeNL** [comment]
This has been solved by #4!
